spec/lib/google_analytics/analytics_import_service_spec.rb

```diff
@@ -1,4 +1,3 @@
-require 'rails_helper'
```

Comment:
I'm a bit wary of this change because it prevents people from writing tests which don't load all of rails (which is why `spec_helper` and `rails_helper` are separate files in the first place).  For example, this analytics spec could be changed to `spec_helper` since it doesn't use any Rails stuff.

The only caveats to this are that a) people often use ActiveSupport stuff (like certain extensions of Ruby objects) in their tests even if those tests don't use Rails otherwise, and b) people have a habit of carelessly using `rails_helper` because it requires less thought, and wind up using it anyway.

This change is not out of the question, but it's very much a capitulation to the idea that we don't encourage lightweight tests, and as a testing philosophy leads over time to slower and slower test runs.

Reply:
I agree with you about this making tests slower and going against encouraging lightweight tests.

However, your caveats are the reason I still think this is a good idea. At the very least, most (if not all) of the tests are using Rails auto-loading which I think only works if you do `require "rails_helper"`. Plus the fact that all the tests are requiring `rails_helper` anyway means that this specific change doesn't actually make the tests any slower.